src/k8s-engine/lima.ts

```diff
@@ -467,9 +495,12 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
       ('hostSocket' in entry));
 
     if (!dockerPortForwards) {
+      const hostSocketPath = `${ paths.lima }/${ MACHINE_NAME }/docker.sock`;
+
+      await this.checkExpandedLength(hostSocketPath);
       config.portForwards?.push({
         guestSocket: '/run/docker.sock',
-        hostSocket:  `{{.LimaHome}}/${ MACHINE_NAME }/docker.sock`,
+        hostSocket:  hostSocketPath,
```

Comment:
I think this is still the original path, and not the one with the expanded symlinks, right? You need to forward to the valid socket path.

Reply:
Ah, I thought we just had to verify the length of the symlink-evaluated path was within limits.